src/collections/pages/pages.11tydata.js

```diff
@@ -1,36 +1,28 @@
 "use strict";
 
-const getLang = require("../../utils/getLang.js");
+const { EleventyI18nPlugin } = require("@11ty/eleventy");
 const generatePermalink = require("../../utils/generatePermalink.js");
 
 module.exports = {
-    layout: "layouts/page.njk",
+    langDir: "ltr",
```

Comment:
is `langDir` used? If so, can we default to `"ltr"` so that it doesn't have to be specified unless it's different?